A/AmazonCACerts/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# This script downloads and bundles CA certificates from Amazon Web Services (AWS).
+# Right now only the RDS root certificate is included, but more could be added in the future.
+
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "AmazonCACerts"
+# Info here: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html
+certificate_version = "2019"
+tarball_version = v"1.0.0"
+
+# Collection of sources required to build AmazonCACerts
+sources = [
+    FileSource("https://s3.amazonaws.com/rds-downloads/rds-ca-$certificate_version-root.pem",
+    "d464378fbb8b981d2b28a1deafffd0113554e6adfb34535134f411bf3c689e73",
+    filename="rds-ca-root.pem"),
```

Comment:
IMO we should keep the version in the file name so people can be aware of which version we're providing, in case we switch in the future. The variable name can remain the same.

Reply:
I don't have a strong opinion on this, but yeah, what you suggest is doable and totally fine.